recipes/sudo-edit

```diff
@@ -0,0 +1,3 @@
+(mv-shell
+ :url "https://github.com/nflath/sudo-edit.git"
```

Comment:
To use `:fetcher github` and `:repo` attribute is better than `:fetcher git` + `:url` attribute.